src/windows/export_clips.py

```diff
@@ -101,28 +101,28 @@ def startAndEndFrames(clip):
 def setupWriter(clip, writer):
     # TODO: allow for audio clips
     # Set video options
-    if clip.data.get('has_video', False):
-        writer.SetVideoOptions(True,
-                               "libx264",
-                               openshot.Fraction(clip.data.get("fps").get("num"),
-                                                 clip.data.get("fps").get("den")),
-                               clip.data.get("width"),
-                               clip.data.get("height"),
-                               openshot.Fraction(clip.data.get("pixel_ratio").get("num"),
-                                                 clip.data.get("pixel_ratio").get("den")),
-                               False,
-                               False,
-                               22
-                                   )
+    pr = clip.data.get("pixel_ratio", {"num": 1, "den": 1})
+    pixel_ratio = openshot.Fraction(pr.get("num"), pr.get("den"))
+    fps = clip.data.get("fps", {"num": 30, "den": 1})
+    frames_per_second = openshot.Fraction(fps.get("num",1), fps.get("den",1))
+
+    writer.SetVideoOptions(True,
+                           "libx264",
+                           frames_per_second,
+                           clip.data.get("width", 1280),
+                           clip.data.get("height", 720),
+                           pixel_ratio,
+                           False,
+                           False,
+                           22)
+    writer.PrepareStreams()
     # Set audio options
-    if clip.data.get('has_audio', False):
-        writer.SetAudioOptions(True,
-                               clip.data.get("acodec", "aac"),
-                               clip.data.get("sample_rate"),
-                               clip.data.get("channels"),
-                               clip.data.get("channel_layout"),
-                               clip.data.get("audio_bit_rate")
-                               )
+    writer.SetAudioOptions(True,
+                           "aac",
```

Comment:
Here was the problem! I was by default using whatever audio encoding the file had. (even when they were mp4 files this caused a crash)